### DISCUSSION TITLE
Conditional columns

### DISCUSSION THREAD

**Discussion by DrewNeon** [filamentphp/filament#10448] [Help] [answered]
How to output different columns of a table according to specific conditions, e.g. by user roles?

**Comment by leandrocfe** [comment, marked as answer]
Use `visible()` or `hidden()`

```php
->visible(fn (): bool => auth()->user()->is_admin)
```

**Comment by DrewNeon** [comment]
> ```
> ->visible(fn (): bool => auth()->user()->is_admin)
> ```
